README.md

```
@@ -69,26 +69,36 @@ func main() {
     if err != nil {
+        // All error messages are always localized and user-friendly
         log.Fatal(err)
     }
     fmt.Printf("Company: %s, CNPJ: %s\n", company.BrazilianCompany.Name, company.BrazilianCompany.CNPJ)
 }
 ```
 
+### Error Messages & Localization
+
+All error messages returned by Mocai are always localized and user-friendly. When an error occurs (e.g., invalid data, unsupported language, or generation failure), the error message will be presented in the language configured for the `Mocker` instance. This ensures that both developers and end-users receive clear, context-appropriate feedback.
+
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)